data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,734 @@
+
+mission "You Are A Pirate"
+	landing
+	source
+		government "Pirate"
+	to offer
+		not "chosen sides"
+		or	
+			and
+				or
+					"reputation: Republic" < -10
+					"reputation: Syndicate" < -10
+					"reputation: Free Worlds" < -10
+				"reputation: Merchant" < -10
+			"reputation: Pirate" > 20
+	on offer
+		conversation "You Are A Pirate"
+	on accept
+		event "became a pirate"
+		fail
+
+conversation "You Are A Pirate"
+	`Preparing to land is second nature by now: patch any perforations in the inner hull, get in touch with ground control, and then make sure your chosen parking pad isn't controlled by extortionists. When you disembark it's with a pistol at your hip, bribe money in your pocket, and a swagger in your step; to leave behind any one of these is to invite trouble. Perhaps a life of violence on the high skies and association with the rougher elements of the galaxy weren't what you had in mind when you left your homeworld, but it's what you found.`
+	choice
+		`	(This is my life now. Piracy, smuggling, the existence of the outlaw. I'm ready to embrace it.)`
+		`	(This isn't what I wanted. This isn't the life I imagined when I decided to take to the high skies.)`
+			decline
+	`	Resolve will carry you far in this world. Tonight it's carried you to a spaceport paint supplier; you return to your vessel with a pallet loaded with blacks and reds. Apply it to your vessel, make your intentions known to the galaxy. There can be no pretense of peace when you fly girded in blood and midnight, but maybe that's exactly the way you want it to be.`
+	choice
+		`	(Don the smock and mask, turn on the compressor, conceal your old life beneath bellicose black.)`
+		`	(No. Not now, perhaps not ever. The galaxy at large may consider you a pirate, but you're not ready to adopt their colors.)`
+			decline
+
+	`	Some paint covers all manner of sin. Some speaks of the intention to commit yet more. You've applied the latter sort to your ship.`
+		accept
+
+
+
+mission "I Want To Be A Pirate"
+	clearance
+	name "Become a pirate"
+	description "Being a pirate in the eyes of the law isn't the same as being one in the eyes of fellow buccaneers. You'll have them know your name - but first you must make one for yourself."
+	source
+		government "Pirate"
+	destination "Greenrock"
+	to offer
```

Comment:
The first mission is if you already grinded out the reputation, already earning the respect. You effectively are already one in that case, so the landing mission just formalizes it and grants you the swizzle. This second mission is for players coming in more as a merchant, but who aspire to be a pirate, looking to get into the pirate life by doing specific favors.

Reply:
Yeah, this one's for greenhorns who've done absolutely nothing wrong (yet) but - for one reason or another - have decided that it's a pirate's life for them.

Reply:
I guess such an idea seems odd to me. "Hello, I'd like to be a pirate, but I'm still on the right side of the law as I haven't actually done anything bad yet!" seems like an unlikely thing to ever happen.

Reply:
Grinding up rep in Endless Sky is a pretty tedious and nebulous task. I would know, I created a pirate file and spent an hour bribing pirates, merchants, and militia in Aldhibain to slowly repair my way into being a pilot. The upper mission is the acknowledgement that someone put in the work, and is more of an easter egg/shortcut for those people. For people coming into the game on newer pilots, looking to be a pirate, this is more of the standard on ramp, offered by just landing on a pirate planet.

Reply:
It is indeed odd, which is why the NPCs react as they do - the player in this is clearly a starry-eyed moonbrain, and they regard your request with confusion and amusement but entertain it nonetheless. 

Yeah. Nobody *wants* to be a pirate. But sometimes people have bigger dreams than sense, and they're the ones that tend to get big things done - if only through dumb luck.

All that aside, it's a Secret of Monkey Island reference.

Reply:
To put it another way, you're a young person from the middle of nowhere with no inheritance and no education, just a dream and a pilot's license.

You won't make it big by working for some company, they tend to want a degree.

You can't just buy your way to the top (yet).

But organized crime? That's a traditional route for people who have nothing more than ambition to rise to heights otherwise inaccessible.

Reply:
It's also meant to lean into the Sparrow's shipyard description, and all those other pirate interceptors you see out in the wild. There needs to be a *relatively* easy path to becoming a pirate, otherwise, how'd all those tiny ships you massacre in the hundreds or thousands get to that point?

Reply:
All right!